packages/mui-joy/src/Autocomplete/Autocomplete.tsx

```diff
@@ -668,12 +670,14 @@ const Autocomplete = React.forwardRef(function Autocomplete(
             }
             return renderListOption(option, index);
           })}
-          {loading && groupedOptions.length === 0 ? (
-            <SlotLoading {...loadingProps}>{loadingText}</SlotLoading>
-          ) : null}
           {groupedOptions.length === 0 && !freeSolo && !loading ? (
             <SlotNoOptions {...noOptionsProps}>{noOptionsText}</SlotNoOptions>
           ) : null}
+
```

Comment:
A convention, we never use line breaks in JSX.

```suggestion
```

Unless it's added automatically.